telegram/_chatpermissions.py

```diff
@@ -121,3 +121,23 @@ def __init__(
             self.can_invite_users,
             self.can_pin_messages,
         )
+
+    @classmethod
+    def all_permissions(cls) -> 'ChatPermissions':
+        """
+        This method returns the ChatPermissions object with all attributes set to :obj:`True`.
```

Comment:
```suggestion
        This method returns an object with all attributes set to :obj:`True`.
```

Reply:
why an undefined an? This is the object where the attribute belongs to... If you dont like repeating ChatPermission, I would vote for either "returns the object" or "returns this object".

Reply:
IMO it's obvious enough that it would return an instance of the class this method is a classmethod of … I'd be okay with ``a :class:`ChatPermisisons` object``

Reply:
so actually, object is not good, since everything in python is an object. But instance sounds great, will do that.